pkg/quota/v1/install/patch_ignoredresources.go

```diff
@@ -0,0 +1,34 @@
+package install
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+func init() {
+	newIgnoredResources := map[schema.GroupResource]struct{}{
+		{Group: "extensions", Resource: "replicationcontrollers"}:                     {},
+		{Group: "extensions", Resource: "networkpolicies"}:                            {},
+		{Group: "", Resource: "bindings"}:                                             {},
+		{Group: "", Resource: "componentstatuses"}:                                    {},
+		{Group: "", Resource: "events"}:                                               {},
+		{Group: "authentication.k8s.io", Resource: "tokenreviews"}:                    {},
+		{Group: "authorization.k8s.io", Resource: "subjectaccessreviews"}:             {},
+		{Group: "authorization.k8s.io", Resource: "selfsubjectaccessreviews"}:         {},
+		{Group: "authorization.k8s.io", Resource: "localsubjectaccessreviews"}:        {},
+		{Group: "authorization.k8s.io", Resource: "selfsubjectrulesreviews"}:          {},
+		{Group: "authorization.openshift.io", Resource: "selfsubjectaccessreviews"}:   {},
+		{Group: "authorization.openshift.io", Resource: "subjectaccessreviews"}:       {},
+		{Group: "authorization.openshift.io", Resource: "localsubjectaccessreviews"}:  {},
+		{Group: "authorization.openshift.io", Resource: "resourceaccessreviews"}:      {},
+		{Group: "authorization.openshift.io", Resource: "localresourceaccessreviews"}: {},
+		{Group: "authorization.openshift.io", Resource: "selfsubjectrulesreviews"}:    {},
+		{Group: "authorization.openshift.io", Resource: "subjectrulesreviews"}:        {},
+		{Group: "authorization.openshift.io", Resource: "roles"}:                      {},
+		{Group: "authorization.openshift.io", Resource: "rolebindings"}:               {},
+		{Group: "authorization.openshift.io", Resource: "clusterroles"}:               {},
+		{Group: "authorization.openshift.io", Resource: "clusterrolebindings"}:        {},
+		{Group: "apiregistration.k8s.io", Resource: "apiservices"}:                    {},
+		{Group: "apiextensions.k8s.io", Resource: "customresourcedefinitions"}:        {},
```

Comment:
@deads2k why do we exclude apiservers and CRDs?

Reply:
> @deads2k why do we exclude apiservers and CRDs?

I have no memory.  They are cluster scoped and not covered by quota.  Should be removeable.

Reply:
As per a slack conversation, we can drop cluster-scoped resources from this list.

Reply:
But trimming the file is deferred till after this lands to not delay things further.